chain/jsonrpc/test-utils/Cargo.toml

```diff
@@ -13,3 +13,7 @@ near-primitives = { path = "../../../core/primitives" }
 near-client = { path = "../../client" }
 near-network = { path = "../../network" }
 near-jsonrpc = { path = "../" }
+near-store = { path = "../../../core/store" }
```

Comment:
I don't see any usage. Should this be cleaned up?

Reply:
removed